src/service/action-impl.js

```diff
@@ -524,12 +535,16 @@ export class ActionService {
   /**
    * @param {!Element} target
    * @param {string} actionEventType
+   * @param {!Element=} opt_parent
```

Comment:
nit: I suggest `opt_stopAt` to match similar methods in `dom.js`. Also makes it more clear that it is exclusive